src/main/java/uk/gov/hmcts/reform/sscs/service/DwpAddressLookupService.java

```diff
@@ -2,8 +2,8 @@
 
 import static java.util.Arrays.stream;
 import static java.util.Objects.nonNull;
+import static java.util.Optional.of;
```

Comment:
This refactoring is a bit like #844. I would rather you based your PR on that PR.